examples/skorch_simple.py

```diff
@@ -0,0 +1,108 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using skorch.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+skorch, and MNIST. We optimize the neural network architecture. As it is too time
+consuming to use the whole MNIST dataset, we here use a small subset of it.
+
+You can run this example as follows, pruning can be turned on and off with the `--pruning`
+argument.
+    $ python skorch_simple.py [--pruning]
+
+"""
+
+import argparse
+
+import numpy as np
+import optuna
+from optuna.integration import SkorchPruningCallback
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+import skorch
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+SUBSET_RATIO = 0.4
+
+mnist = fetch_openml("mnist_784", cache=False)
+
+X = mnist.data.astype("float32")
+y = mnist.target.astype("int64")
+indices = np.random.permutation(len(X))
+N = int(len(X) * SUBSET_RATIO)
+X = X[indices][:N]
+y = y[indices][:y]
```

Comment:
I'm not sure, but
```suggestion
y = y[indices][:N]
```
?